api/base/serializers.py

```diff
@@ -786,6 +786,23 @@ def to_representation(self, value):
         return relationship
 
 
+class TypedRelationshipField(RelationshipField):
```

Comment:
Discussed on FD: document the assumption that the url namespace MUST be the same as the JSONAPI type.

Reply:
Added.